lms/static/js/fixtures/search_form.html

```diff
@@ -0,0 +1,7 @@
+<div id="courseware-search-bar" class="search-container">
+    <form role="search-form">
+        <input type="text" class="search-field"/>
+        <button type="submit" class="search-button" aria-label="Search">search <i class="icon-search"></i></button>
+        <button type="button" class="cancel-button" aria-label="Cancel"><i class="icon-remove"></i></button>
```

Comment:
@mattdrayer, @chrisndodge, @cdodge, keep the aria-label here, since there is no accessible text, but add an aria-hidden to the iconfont:
```<button type="button" class="cancel-button" aria-label="Cancel"><i class="icon-remove" aria-hidden="true"></i></button>```